### DISCUSSION TITLE
light-blocking material

### DISCUSSION THREAD

**Discussion by cloudziq** [redeclipse/base#1116] [Ideas and Feature Requests]
I know that Tesseract have 'nogi' material, which it's (still) not ported to RE.
Can we have a similiar material that blocks lighting only?  
In some cases, to improve performance and overall visuals on a map, mappers can use no-dynamic, ambient light (with flag=3), which illuminate what is behind some obstacles. Material like that I suggest can help with this, because just reducing light radius is sometimes out of question (so mappers need to use a few light entities rather than a one - and effect is way worse, sometimes impossible to achieve). Using a few lights with flag=2 is a bad thing to do, in visuals terms too (where we dont need shadows at all). Hope this is possible to do and can help much with porting existing maps or when creating a new ones

**Comment by Calinou** [comment]
Tesseract's `nogi` material only blocks *indirect* lighting (i.e. what is generated by global illumination, which is enabled by default and can be disabled for debugging purposes using `/gi 0`).

**Comment by SniperGoth** [comment]
He already knows what /nogi does, Calinou.
What he is asking is about a sort of /nolight volume (yeah, i call materials volumes, for reasons), which does the same thing, but only for point lights/spotlights in this case.
